src/sentry/stacktraces/processing.py

```diff
@@ -33,6 +33,9 @@ def __eq__(self, other: object) -> bool:
     def __ne__(self, other: object) -> bool:
         return self is not other
 
+    def get_frames(self) -> Sequence[dict[str, str]]:
+        return get_path(self.stacktrace, "frames", filter=True, default=())
```

Comment:
Total nit: If normally this gives back a list, is there a reason to make the default a tuple rather than a list?